pkg/controllers/provisioning/scheduling/suite_test.go

```diff
@@ -3704,7 +3704,7 @@ var _ = Context("Scheduling", func() {
 			Expect(err).To(BeNil())
 			s.Solve(injection.WithControllerName(ctx, "provisioner"), pods)
 
-			m, ok := FindMetricWithLabelValues("karpenter_provisioner_scheduling_simulation_duration_seconds", map[string]string{"controller": "provisioner"})
+			m, ok := FindMetricWithLabelValues("karpenter_scheduler_duration_seconds", map[string]string{"controller": "provisioner"})
```

Comment:
```suggestion
			m, ok := FindMetricWithLabelValues("karpenter_scheduler_scheduling_duration_seconds", map[string]string{"controller": "provisioner"})
```

I think we should change this name to be `karpenter_scheduler_scheduling...` if we can